ui/components/Link.tsx

```diff
@@ -58,6 +58,6 @@ function Link({
 export default styled(Link)`
   text-decoration: none;
   &.title-bar-button {
-    width: 250px;
+    width: 240px;
```

Comment:
What is the reason for changing the styling of our custom Link component if we are directly importing the Link component from the React Router in the Logo component?

Reply:
Great point - I think I'm able to delete that class entirely now as it *should* be unused. Will verify.

Reply:
Update - definitely unused - pushed up new code removing the class and updating the snaps